chart/templates/cleanup/cleanup-serviceaccount.yaml

```diff
@@ -21,6 +21,9 @@
 {{- if and .Values.cleanup.serviceAccount.create .Values.cleanup.enabled }}
 kind: ServiceAccount
 apiVersion: v1
+{{- if hasKey .Values.cleanup.serviceAccount "automountServiceAccountToken" }}
```

Comment:
Not all services allow you to specify additional parameters for serviceAccount for example cleanup. You should alter chart/values.schema.json. Also, I think it's better to reflect the new available parameter with a default value in the chart/values.json, because it's not clear where I should specify it.